rtsp.go

```diff
@@ -457,6 +457,10 @@ func (rc *rtspCamera) initH265(session *description.Session) (err error) {
 
 		for _, nalu := range au {
 			frame, err := rc.rawDecoder.decode(nalu)
+			var recoverableErr *RecoverableAVError
+			if errors.As(err, &recoverableErr) {
+				continue
```

Comment:
Is there a continue forever case here? What's the worst thing that could happen if we never return from this loop?